frontend/src/index.tsx

```diff
@@ -197,6 +203,22 @@ const Store: Component = () => {
     db.remove_webxdc(app_id)
   }
 
+  const supportsDraggingOut = !!(window as any).webxdc_custom?.desktopDragFileOut
+  const onDragStart = async (ev: DragEvent, item: AppInfoWithState)=>{
+    ev.preventDefault()
```

Comment:
Maybe only prevent default when dragging out is actually used? (e.g move this line one down)